docs/secure-connections/sauce-connect/setup-configuration/ci-cd-integration.md

```diff
@@ -4,6 +4,126 @@ title: Sauce Connect Proxy CI/CD Integration
 sidebar_label: CI/CD Integration
 ---
 
+## Sauce Connect Docker Container Setup
+
+As an alternative to downloading and installing the SC Client in your CI/CD environment, you can [leverage Docker containers to manage Sauce Connect Proxy tunnels](/secure-connections/sauce-connect/installation/#running-sauce-connect-in-docker). Our Docker image maintained by the Sauce Labs [Open Source Program Office](https://opensource.saucelabs.com/).
+
+Here are some benefits/use cases:
+* If you want to run Sauce Connect Proxy as part of a Dockerized CI/CD.
+* If you'd prefer to manage Docker image tags instead of Sauce Connect Proxy versions.
+* If your setup involves several instances running on the same system, Docker would simplify Sauce Connect Proxy port management.
+
+### Running the SC Docker Image
+
+1. Before running the container, you'll need to pull the Sauce Connect Proxy Docker Image from the Docker Hub. This will always pull the latest version of Sauce Connect Proxy.
+  ```bash
+  $ docker pull saucelabs/sauce-connect
+  ```
+   * Or - if you _do_ want to use specific SC version - you can specify that as a tag:
+   ```bash
+   $ docker pull saucelabs/sauce-connect:4.7.1
+   ```
+2. To run the Sauce Connect Proxy Docker image, execute the below script, which will also set your Sauce Labs username and access key as [environment variables](/basics/environment-variables/).
+  ```bash
+  $ export SAUCE_USERNAME="my-user"
+  $ export SAUCE_ACCESS_KEY="my-access-key"
+  docker run \
+      -e SAUCE_USERNAME=${SAUCE_USERNAME} \
+      -e SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY} \
+      -it saucelabs/sauce-connect
+  ```
+
+If desired, you can specify any additional [SC CLI arguments](/dev/cli/sauce-connect-proxy/) here.
+
+::::note
+In Linux, if your tests use localhost addresses, `--network="host"` is required as an argument in the above script to allow requests originating within the container to access the Docker host machine.
+::::
+
+::::warning
+`--network="host"` behaves differently on Windows, Mac OS, and Linux due to the respective platform Docker implementation.
+::::
```

Comment:
We reserve **Warning** admonition boxes for major alerts that could cause data loss or a breaking change

```suggestion
:::info
`--network="host"` behaves differently on Windows, macOS, and Linux due to the respective platform Docker implementation.
:::
```